l10n_es_vat_prorate/models/prorate_taxes.py

```diff
@@ -0,0 +1,166 @@
+PRORATE_TAXES = {
+    "account_tax_template_p_iva21_bc": {
+        "template_with_vat_prorate": True,
+        "prorate_account_template_ids": [],
```

Comment:
No declares esto en los que esté vacío, e igualmente no añadas una clave `template_with_vat_prorate: True`. El hecho de estar el impuesto en este diccionario ya hace que sea con prorrata. Solo tienes que hacer `if <xml_id> in PRORRATE_TAXES`, y luego `PRORRATE_TAXES[<xml_id>].get("prorate_account_template_ids", [])`

Reply:
Insisto en que aquí no debería ser necesario poner `"prorate_account_template_ids": [],` para cada línea. Solo incluye la clave `"account_tax_template_p_iva21_bc": {}` y compruebas con el operador `in` si está incluido el xml-id en este diccionario.